app/controllers/AddASettlorController.scala

```diff
@@ -45,26 +44,27 @@ class AddASettlorController @Inject()(
                                        addAnotherFormProvider: AddASettlorFormProvider,
                                        val controllerComponents: MessagesControllerComponents,
                                        addAnotherView: AddASettlorView,
-                                       yesNoView: AddASettlorYesNoView,
-                                       requiredAnswer: RequiredAnswerActionProvider
+                                       yesNoView: AddASettlorYesNoView
                                      )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport with Enumerable.Implicits {
 
   private val addAnotherForm: Form[AddASettlor] = addAnotherFormProvider()
   private val yesNoForm: Form[Boolean] = yesNoFormProvider.withPrefix("addASettlorYesNo")
 
   private def actions(draftId: String): ActionBuilder[RegistrationDataRequest, AnyContent] =
-    standardActions.authWithData(draftId) andThen requiredAnswer(RequiredAnswer(KindOfTrustPage))
```

Comment:
Is this not required? looks like this answer is needed as part of the trustHintText method

Reply:
@BriWak the `trustHintText` method maps over a get of KindOfTrustPage so will return `None` for a non-taxable trust which is what we want